wcfsetup/install/files/lib/data/user/UserAction.class.php

```diff
@@ -1004,12 +1028,13 @@ public function validateResendActivationMail() {
 			throw new PermissionDeniedException();
 		}
 		
-		if (REGISTER_ACTIVATION_METHOD != 1) {
+		if (!(REGISTER_ACTIVATION_METHOD & User::REGISTER_ACTIVATION_USER)) {
```

Comment:
May-be there should be static methods be added to `UserRegistrationUtil` that check for these scenarios?

Reply:
I don't feel it's necessary. What advantage would it have? Don't you find the current version readable enough? In the template I could check the whole thing with (unnecessary) workarounds and still have to go back to the bit comparison there.